pkg/iam/iam.go

```diff
@@ -118,10 +118,12 @@ func (s *IAMService) ReconcileRole() error {
 	params := struct {
 		ClusterName      string
 		EC2ServiceDomain string
+		AWSDomain        string
```

Comment:
nitpick: why don't we call it `AWSPartition`?

Reply:
I can refactor, should be pretty easy

Reply:
I won't block because of that. If it's easy, let's change it, otherwise, let's keep it.